dlt/dlt_broker.py

```diff
@@ -101,14 +120,17 @@ def __init__(
 
     def start(self):
         """DLTBroker main worker method"""
-        logger.debug(
-            "Starting DLTBroker with parameters: use_proxy=%s, ip_address=%s, port=%s, filename=%s, multicast=%s",
-            False,
-            self._ip_address,
-            self._port,
-            self._filename,
-            ip.ip_address(self._ip_address).is_multicast,
-        )
+        if self._from_dlt_client:
+            logger.debug(
+                "Starting DLTBroker with parameters: use_proxy=%s, ip_address=%s, port=%s, filename=%s, multicast=%s",
+                False,
+                self._ip_address,
+                self._port,
+                self._filename,
+                ip.ip_address(self._ip_address).is_multicast,
+            )
+        else:
+            logger.debug("Starting DLTBroker by reading %s", self._filename)
```

Comment:
You can add `show_init_info` in DLTMessageHandler and DLTFileSpinner then call it to avoid using the if-else

Reply:
sorry, this is the log for DLTBroker, not for dlt message handler.